plugins/contracts/src/generators/generateNamespace.ts

```diff
@@ -81,3 +83,32 @@ function extractNamespaces(
     ),
   }
 }
+
+function throwErrorForDuplicateNames(generatorNamespace: GeneratorNamespace, context: ContractsContext) {
+  const interfaceNames = generatorNamespace.interfaces.map(i => i.getName(context.nameTransform))
```

Comment:
```suggestion
  const interfaceNames = generatorNamespace.interfaces.map(interface => interface.getName(context.nameTransform))
```

IMO there is no reason to shorten it